src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -105,11 +105,11 @@ end
 
 """
     NetCDFOutputWriter(model, outputs; interval=nothing, frequency=nothing, filename=".",
-                                   clobber=true, global_attributes=Dict(), output_attributes=nothing, slice_kw...)
+                                   clobber=false, global_attributes=Dict(), output_attributes=nothing, slice_kw...)
 
-Construct a `NetCDFOutputWriter` that writes `label, field` pairs in `outputs` (which can be a
-`Dict` or `NamedTuple`) to a NC file, where `label` is a symbol that labels the output and
-`field` is a field from the model (e.g. `model.velocities.u`).
+Construct a `NetCDFOutputWriter` that writes `label, field` pairs in `outputs` (which should
+be a `Dict`) to a NC file, where `label` is a symbol that labels the output and `field` is
```

Comment:
why can't we handle named tuples... ?

Reply:
Would be easy to add support for named tuple. I just removed them from the docstring because of https://github.com/climate-machine/Oceananigans.jl/issues/553

We can add support for named tuples in this PR as well.